docker.test.yml

```diff
@@ -11,7 +11,7 @@ sut:
       for i in $$(brew list | egrep -vx 'jdk'); do
         brew linkage --test $$i || failed=1
       done
-      for i in $$(brew list | egrep -vx 'idba|pigz|raxml-ng|treepl|unicycler'); do
+      for i in $$(brew list | egrep -vx 'idba|pigz|raxml-ng|treepl|unicycler|magic-blast|tbl2asn|swi-prolog'); do
```

Comment:
These are `brew linkage` failures and so need to be added to the list on line 11, with `jdk`.